lib/menu.rb

```diff
@@ -0,0 +1,22 @@
+class Menu
+  attr_reader :dishes
+
+  def initialize
+    @dishes = {
+      "Margherita" => 5.00,
+      "Pepperoni" => 8.00
+    }
+  end
+
+  def view_menu
+    @dishes.map do |dish, price|
+      "#{dish}: £#{('%.2f' % price)}"
```

Comment:
This could be refactored onto one line of code.

Reply:
Amended now.